src/packaging/src-dist/MANIFEST.yaml

```diff
@@ -358,8 +354,6 @@ tarballs:
             - thread-registry.c
```

Comment:
Do perl tests pass in you remove the sysfs files from src-dist/MANIFEST.yaml? I though we used those rpms to run our perl tests. I rather though you'd have to exclude these files from the github and kpatch manifests explicitly.